src/pages/ProfilePage.js

```diff
@@ -178,6 +184,20 @@ function ProfilePage(props) {
                                     {displayName}
                                 </Text>
                             )}
+                            {hasStatus && (
```

Comment:
Let's hide this behind the beta as well (and all other PRs). We don't want internal people setting a status, then having external people see it and be confused what it is.

Reply:
👍🏻